app.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser");
+const axios = require("axios");
+
+const port = 3000;
+
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+const animals = ["tiger", "lion", "bear", "seal", "giraffe", "elephant", "wolf", "zebra", "horse", "cow", "shark"];
+
+const isAnimal = (req, res, next) => {
+    if (animals.includes(req.params.species)) {
+        res.json({status: "success", message: true});
+        next();
```

Comment:
There wouldnt be a need to use `next()` since you're already sending a response before hand